translations/messages+intl-icu.de.yml

```diff
@@ -2016,6 +2016,9 @@ maplayer.locked: "Kartenebene gesperrt"
 maplayer.show: "Kartenebene einblenden"
 maplayer.show/hide: "Kartenebenen ein/ausblenden"
 maplayer.capabilities.fetch.error: "Beim Laden der Ressourcen ist ein Fehler aufgetreten. Bitte überprüfen Sie Ihre Eingaben."
+maplayer.capabilities.fetch.error.cors.policy: |
+    Falls das Problem weiterhin auftritt, wenden Sie sich bitte an den Betreiber des Dienstes mit folgendem Hinweis:
+    Es soll sichergestellt werden, dass ein CORS-Header für den Dienst gesetzt ist. Dies wird von der GDI-DE im Kapitel 4.7.1 empfohlen.
```

Comment:
```suggestion
maplayer.capabilities.fetch.error.cors.policy: |
    Der angegebene Dienst konnte nicht geladen werden. Bitte stellen Sie sicher, dass die URL richtig angegeben wurde.
    
    Falls das Problem weiterhin auftritt, wenden Sie sich bitte an den Betreiber des Dienstes mit folgendem Hinweis:
    Es soll sichergestellt werden, dass ein CORS-Header für den Dienst gesetzt ist. Dies wird von der GDI-DE im Kapitel 4.7.1 empfohlen.
```

Reply:
I thought that it would be more readable to have two messages instead of one. The first message informs that something went wrong, please check the inputs. The second message should instruct to contact the map server if the problem occurs again...

![image](https://github.com/demos-europe/demosplan-core/assets/110987766/fd4f9db5-600f-4716-9f7f-8232e15752df)
![image](https://github.com/demos-europe/demosplan-core/assets/110987766/cebe5b64-1ccb-4113-9c79-1909c228c607)

Reply:
I think we should make one message, but with a blank line.  (you can use `<br />`)

Reply:
I used the YAML block scalar syntax with a literal style indicator (` | `). It allows preserving line breaks and indentation. But in this case, it does not work as expected, as well as if I add a `br` tag..

![image](https://github.com/demos-europe/demosplan-core/assets/110987766/d19d3435-f2d2-4139-bb02-865723cbd259)

It could be achieved with the `DpInlineNotification`, but it may not be displayed as an error message on the right side below

Reply:
Ok. I see. I was sure that it works within the flash-messages :-/ Then letzt take Your approach with two messages. But then I suggest to make the second one as warning-Level